cfgov/unprocessed/css/molecules/form-field-with-button.less

```diff
@@ -31,33 +31,37 @@
     - cfgov-molecules
 */
 
-
-
 // TODO: Look into merging email-signup and form-field-with-button styles
 // Form field with button isn't used anywhere else
 .m-form-field-with-button {
     .m-form-field {
-        margin-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
+        margin-bottom: 0;
     }
 
-    &_info {
-        margin-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
+    &_info,
+    .a-btn {
+        margin-top: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
     }
 
     &__inline-info {
-        width: 100%;
-        display: flex;
-        align-items: flex-start;
-        flex-wrap: wrap-reverse;
-        justify-content: space-between;
+        .respond-to-min(321px, {
+            .m-form-field-with-button_wrapper {
+                width: 100%;
+                display: flex;
+                align-items: flex-start;
+                flex-wrap: wrap-reverse;
+                justify-content: space-between;
+            }
 
-        .m-form-field-with-button_info {
-            order: 1;
-        }
+            .m-form-field-with-button_info {
+                order: 1;
```

Comment:
Nice! It's good to see more flexbox layouts being used to keep things in the proper markup order.

Reply:
Yeah, I'm always impressed at how powerful flexbox is for things like this!